styles.css

```diff
@@ -0,0 +1,476 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+
+:root {
+  --black: #272a31;
+  --orange: #ec5242;
+  --grey: #d3d3d3;
+  --mainFont: 'Lato', sans-serif;
+}
+
+html,
+body {
+  box-sizing: border-box;
+  font-family: var(--mainFont);
+  padding: 0;
+  margin: 0;
+  position: relative;
+  z-index: 10;
+}
+
+.display-none {
+  display: none;
+}
+
+.container {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
+.modalcpst {
+  background-color: rgba(236, 82, 66, 0.9);
+  height: 100vh;
+  position: absolute;
+  z-index: 4;
+  width: 100%;
+}
+
+.modal__navigation ul {
+  list-style: none;
+  padding-left: 0;
+}
+
+.modal__navigation__li {
+  margin-bottom: 1.8rem;
+}
+
+.modal__navigation__li a {
+  color: var(--grey);
+  text-transform: capitalize;
+  font-size: 2.3rem;
+  text-decoration: none;
+}
+
+.modal__exit-button {
+  color: var(--grey);
+  display: flex;
+  justify-content: end;
+  padding: 25px 0;
+}
+
+.stop-scrolling {
+  overflow: hidden;
+}
+
+.topPageDiv {
+  display: none;
+}
+
+.topPage__img {
+  padding: 7px;
+}
+
+.topPage__text {
+  padding: 1px 5px;
+}
+
+.navbar {
+  display: none;
+}
+
+.hamburger {
+  padding: 10px;
+}
+
+.line {
+  width: 25px;
+  height: 5px;
+  background-color: var(--grey);
+  border-radius: 7px;
+  margin: 5px;
+}
+
+.logo {
+  width: 40px;
+}
+
+.nav-link {
+  color: var(--grey);
+}
+
+header {
+  background-image: url(./elements/bckgrnd8.jpeg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  height: 100%;
+}
+
+.main-text {
+  padding: 15% 2% 15% 1%;
+}
+
+.main-text__title {
+  background-image: url(./elements/bckgrndalpha.svg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 46px;
+  text-transform: uppercase;
+  font-weight: 700;
+  -webkit-font-smoothing: antialiased;
+}
+
+.main-text__bio {
+  color: var(--grey);
+}
+
+.main-text__dates,
+.main-text__location {
+  color: var(--grey);
+  font-weight: bolder;
+}
+
+.main-text__location {
+  font-weight: 300;
+  font-size: 30px;
+}
+
+#about {
+  background-image: url(./elements/bckgrnd8.jpeg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
```

Comment:
- [ ] As requested before it is required to use the same background as the original design :+1: . [Here](https://ingdavidtm.github.io/capstone-biotechnology/images/backgroundHeaderAbout.png) you can find the required background.

**Your design**

![Screenshot from 2023-02-09 06-34-44](https://user-images.githubusercontent.com/105117832/217802099-cdba4c87-d84d-4c0d-bb24-31c063513084.png)

**Template**

![Screenshot from 2023-02-07 04-46-49](https://user-images.githubusercontent.com/105117832/217215516-59593994-9983-40cd-adf3-62497dbf6278.png)